internal/printer/printer_test.go

```diff
@@ -198,15 +206,14 @@ const name = "world";
 
 <body>
   <!-- your content here... -->
-  <script src="js/scripts.js"></script>
```

Comment:
This was missing and I don’t know where it went, but I think it’s part of the script hoisting work and I’m not too worried about it. If this is a bug it won’t be one for long.